packages/primitives/src/Svg/SvgIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Svg } from './Svg'
+
+type SvgIconProps = {
+  title?: string
+  desc?: string
+} & React.ComponentProps<typeof Svg>
+
+/* Adds accesibility to the Svg element */
+export const SvgIcon = ({ title, desc, ...props }: SvgIconProps) => (
+  <Svg {...props}>
+    <title>{title}</title>
+    <desc>{desc}</desc>
+    {props.children}
+  </Svg>
+)
```

Comment:
@alexanderchan we need to allow for the setting of the viewbox, height, and width here.

Reply:
Yup, this comes from the Svg and we pass the props down to it, the type has the same has Svg and we add in title and desc.  I've added them to the SvgIcon spec so we can see it in the snapshot.

```tsx
type SvgIconProps = {
  title?: string
  desc?: string
} & React.ComponentProps<typeof Svg>
```

Reply:
Oh duh, good point! Thank you for clarifiing!